sklearn/tests/test_pipeline.py

```diff
@@ -222,6 +223,20 @@ def test_pipeline_init_tuple():
     pipe.score(X)
 
 
+def test_pipeline_init_dict():
+    # Pipeline accepts steps as items of dict
+    X = np.array([[1, 2]])
+    pipeline_estimators = {'transf': Transf(),
+                           'clf': FitParamT()}
+    pipe = Pipeline(pipeline_estimators.items())
```

Comment:
what does that test? That we allow ``items`` in pipelines? Do we want to guarantee that?